comprobo_final_project/scripts/gene_trainer.py

```diff
@@ -50,7 +50,8 @@ def train(self):
                 file_obj.flush()
 
                 if self.population.generations[0].fitness < 0.05:
```

Comment:
the fitness end condition will need to change, if the fitness is now the average. I would think there's almost no way a robot's average distance away from the goal could be .05, although i'm not sure.